Dockerfile.dev

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11.5-bookworm
+
+RUN mkdir SlashGPT
+WORKDIR /SlashGPT
+
+# RUN git clone https://github.com/snakajima/SlashGPT.git
+
+WORKDIR /SlashGPT/SlashGPT
+
+ARG REQUIREMENTS=requirements/full.txt
+
+#COPY requirements.txt requirements.txt
+COPY $REQUIREMENTS requirements.txt
+RUN pip install -r requirements.txt
+RUN pip install playsound
+COPY .env .env
```

Comment:
This is not necessary if you mount with docker -v

If you don't mount it, you need the clone above